### PR TITLE
Yarn create

### PR DESCRIPTION
**Pull request by arcanis**
cc @cpojer @bestander - also cc @gaearon since create-react-app would probably be interested

### CODE REVIEW

**Comment by bestander** [comment]
I think it is a good idea.
Would it work with most common bootstrappers (react, react native, ember, angular) out of the box?

**Comment by arcanis** [comment]
I think so, maybe with two small restrictions:

- Semantically speaking, it would probably be for the best if they could support the `yarn create <package-name>` command without requiring extra parameter (for example, `ng new` doesn't seem to work, but `ng new foobar` does). I mean, it would work since we would forward the extra arguments to the create script anyway, but it would only fragment the syntax.

- They would have to support generating files into another directory than the current one (or they can do `(cd $npm_create_path && ng new)`, but it might not work on Windows ... I'm not sure about this one)

File: text/0000-yarn-create.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+# Detailed design
+
+This RFC suggests to add a new command:
+
+```
+$> yarn create <package-name>
+```
+
+Running this command would have the samne effect as:
```
Comment (cpojer) [inline text/0000-yarn-create.md]: "same"

File: text/0000-yarn-create.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+```
+$> yarn add --dev <package-name>
```
Comment (cpojer) [inline text/0000-yarn-create.md]: should be `yarn global add`, right?

Comment (arcanis) [inline text/0000-yarn-create.md]: Maybe not, cf the third point https://github.com/arcanis/rfcs/blob/c6afb08a2a516c5dca4d03429c31bd8f91a9a03f/text/0000-yarn-create.md#alternatives

**Comment by arcanis** [comment]
I've just published a tentative implementation on https://github.com/yarnpkg/yarn/pull/3244.

**Pull request merged**